sphinx/source/conf.py

```diff
@@ -105,13 +105,21 @@
 
 import jpy
 py_scope_jpy = jpy.get_type("io.deephaven.engine.util.PythonScopeJpyImpl").ofMainGlobals()
+
+no_op_thread_factory = jpy.get_type("io.deephaven.util.thread.ThreadInitializationFactory").NO_OP
+_JOperationInitializationThreadPool = jpy.get_type("io.deephaven.engine.table.impl.OperationInitializationThreadPool")
+_j_operation_initializer = _JOperationInitializationThreadPool(no_op_thread_factory)
+
 _JUpdateGraph = jpy.get_type("io.deephaven.engine.updategraph.impl.PeriodicUpdateGraph")
-docs_update_graph = _JUpdateGraph.newBuilder("PYTHON_DOCS").build()
+docs_update_graph = _JUpdateGraph.newBuilder("PYTHON_DOCS") \
```

Comment:
Same comment as on the test helper. Maybe consult with @jmao-denver to make sure we have the right conventions.

Reply:
these arguably could be `_` prefixed... but this is configuration for running sphinx, not something that users can use (or that we would import anywhere else), so shouldn't matter.

Reply:
+1, and the same comment above applies here too.